kubespawner/reflector.py

```diff
@@ -27,6 +28,15 @@ class ResourceReflector(LoggingConfigurable):
     kubernetes resources.
 
     Must be subclassed once per kind of resource that needs watching.
+
+    Creating a reflector should be done with the create() classmethod,
+    since that, in addition to creating the instance, initializes the
+    Kubernetes configuration, acquires a K8s API client, and starts the
+    watch task.
```

Comment:
Hmmm...  I wonder if we may want to make this a `create()` be `async_init_and_start()` or similar?

```suggestion
    Creating a reflector should be done with the create() classmethod,
    since that, in addition to creating the instance starts the watch task.
```